node/bin/src/main.rs

```diff
@@ -303,6 +316,27 @@ fn build_external_config(repo: ConfigRepository<'_>) -> Config {
     }
 }
 
+fn enable_sandbox_mode(config: &mut Config) -> Option<TempDir> {
+    let original_path = config.general_config.rocks_db_path.clone();
+    if original_path != Path::new(DEFAULT_ROCKS_DB_PATH)
+        || std::env::var_os("GENERAL_ROCKS_DB_PATH").is_some()
```

Comment:
Nit: no need to check the envvar directly, `config.general_config.rocks_db_path` gets populated from it anyway (that's how `smart-config` works)

Reply:
Makes sense! Removed env var check.